### PR TITLE
Fix disabled operation metadata

### PR DESCRIPTION
**Pull request by soyuka**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | https://github.com/api-platform/core/pull/3079
| License       | MIT
| Doc PR        | https://github.com/api-platform/docs/pull/886

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -89,7 +89,8 @@
     },
     "conflict": {
         "doctrine/common": "<2.7",
-        "doctrine/mongodb-odm": "<2.0"
+        "doctrine/mongodb-odm": "<2.0",
+        "doctrine/mongodb-odm-bundle": "4.0.0-RC2"
```
Comment (teohhanhui) [inline composer.json]: We shouldn't do this...

Comment (alanpoulain) [reply composer.json]: We will forget it.

Comment (teohhanhui) [reply composer.json]: And it only conflicts with a specific version so when RC3 is out it'll break again. :laughing:

Comment (alanpoulain) [reply composer.json]: RC3 should resolve the issue so I don't think it will break.

Comment (teohhanhui) [reply composer.json]: Oh, okay. If it's a bug in the bundle, then I'm kind of fine with this. We can remove this conflict when RC3 is released.

Comment (teohhanhui) [reply composer.json]: Anyway, this commit doesn't belong here. :laughing:

Comment (soyuka) [reply composer.json]: it's a cherry pick to make things green :)

**Pull request closed without merging**